spec/polymorphism_spec.rb

```diff
@@ -4,34 +4,51 @@
   include JsonHelpers
   include_context 'resource testing'
 
-  let(:resource) { Class.new(PORO::CreditCardResource) }
+  # Inheriting causes us to think this class is a polymorphic
+  # child. Let it know this is not so, we just want a subclass for testing
+  let(:resource) do
+    Class.new(PORO::CreditCardResource) do
+      self.polymorphic_child = false
```

Comment:
Seems we are limiting ourselves to only one level of inheritance by using a boolean. Polymorphic Parent with a reference to the parent resource (nil for top level) would allow this behavior even on child resources.

Reply:
The DSL only allows for one level...I'm not sure multiple levels would ever be needed/possible. Not sure I'm totally understanding how this would work.

Reply:
Ok if we only support one level then that's fine.  Just thought this was limiting us to only using this `Class.new(Foo)` thing for extending the parent class.